_maps/map_files/tramstation/tramstation.dmm

```diff
@@ -32473,7 +32473,7 @@
 /turf/open/floor/iron,
 /area/station/maintenance/tram/mid)
 "kxF" = (
-/obj/item/kirbyplants/dead,
+/obj/item/kirbyplants/dead/rd,
```

Comment:
Should we add the RD's potted plant to the unit test for things that should be in all maps?